docs/source/markdown/options/file.md

```diff
@@ -0,0 +1,16 @@
+####> This option file is used in:
+####>   podman build, farm build
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+#### **--file**, **-f**=*Containerfile*
+
+Specifies a Containerfile which contains instructions for building the image,
+either a local file or an **http** or **https** URL.  If more than one
+Containerfile is specified, *FROM* instructions are only be accepted from the
```

Comment:
_are only be accepted_
(is that correct English?)

Reply:
no, the word "be" should be removed here.  Nice spot!